From 4da2c88fd856718599472b9f2d34b0182e6ee3ba Mon Sep 17 00:00:00 2001
From: Aelin <aelin@postmarketos.org>
Date: Tue, 14 Apr 2026 09:31:54 +0200
Subject: [PATCH 02/10] Allow systemd-oomd to log to syslog

Resolves the following denial:
[ 9.285187] audit: type=1400 audit(1776151460.879:4): avc: denied { sendto } for pid=348 comm="systemd-oomd" path="/run/systemd/journal/socket" scontext=system_u:system_r:systemd_oomd_t:s0 tcontext=system_u:system_r:syslogd_t:s0 tclass=unix_dgram_socket permissive=1
---
 policy/modules/system/systemd.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index 2f626f122..1f323ddb6 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -2079,6 +2079,8 @@ fs_list_cgroup_dirs(systemd_oomd_t)
 fs_setattr_cgroup_dirs(systemd_oomd_t)
 fs_write_cgroup_dirs(systemd_oomd_t)
 
+logging_send_syslog_msg(systemd_oomd_t)
+
 optional_policy(`
     dbus_acquire_svc_system_dbusd(systemd_oomd_t)
     dbus_read_pid_sock_files(systemd_oomd_t)
-- 
2.55.0

